.github/workflows/ci.yml

```diff
@@ -644,7 +644,7 @@ jobs:
         # Run tests if any of the secrets is present. Do not skip tests when one secret renamed, or secret name has a typo.
         if: >-
           contains(matrix.modules, 'trino-delta-lake') && contains(matrix.profile, 'cloud-tests') &&
-          (env.ABFS_ACCOUNT != '' || env.ABFS_CONTAINER != '' || env.ABFS_ACCESSKEY != '' || env.AWS_ACCESS_KEY_ID != '' || env.AWS_SECRET_ACCESS_KEY != '' || env.GCP_CREDENTIALS_KEY != '')
+          (secrets.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || env.ABFS_ACCOUNT != '' || env.ABFS_CONTAINER != '' || env.ABFS_ACCESSKEY != '' || env.AWS_ACCESS_KEY_ID != '' || env.AWS_SECRET_ACCESS_KEY != '' || env.GCP_CREDENTIALS_KEY != '')
```

Comment:
We already have `SECRETS_PRESENT`

Reply:
I'm not sure if I understand the difference.

Reply:
SECRETS_PRESENT controls whether to upload heap dumps.
any fork/downstream project will want to define that

CI_SKIP_SECRETS_PRESENCE_CHECKS is an opt-in -- for use in Trino project and maybe also others -- to be used whenever there is intention to run all secrets-requiring tests